src/screens/auth/SignUp.tsx

```diff
@@ -30,29 +29,19 @@ const Signup: React.FC = () => {
         <form>
           <VStack align="stretch">
             <FormControl>
-              <FlotingInput label="First Name" name="firstname" />
-              <FlotingInput label="Last Name" name="lasttname" />
-              <FlotingInput label="Mobile Number" name="mobilenumber" />
-              <FloatingPasswordInput
-                label="Create Password"
-                name="createpassword"
-              />
-              <FloatingPasswordInput
-                label="Confirm Password"
-                name="confirmpassword"
-              />
+              <FlotingInput label="First Name" />
```

Comment:
can these labels come from **lang** file?

Reply:
No for now labels are not coming from lang file once API integration gate completed @sonaliTekdi and i will work on that